cli/lib/kontena/cli/plugins/install_command.rb

```diff
@@ -0,0 +1,27 @@
+require 'open3'
+
+module Kontena::Cli::Plugins
+  class InstallCommand < Clamp::Command
+
+    parameter 'NAME', 'Plugin name'
+
+    def execute
+      install_plugin(name)
+    end
+
+    def install_plugin(name)
+      plugin = "kontena-plugin-#{name}"
+      gem_bin = `which gem`.strip
```

Comment:
We need to get the full path because I did not find any solid way to pass environment. This is a problem with rvm and friends that modify env variables.